src/Folder.jsx

```diff
@@ -1,28 +1,104 @@
+import { useState, useEffect } from 'react';
+
+import { getFolderList, getLink, getLinkDetail } from './apis/api';
+import shareImg from './assets/folder/share.svg';
+import renameImg from './assets/folder/pen.svg';
+import deleteImg from './assets/folder/delete.svg';
+
+import FolderAddBtn from './components/FolderAddBtn';
 import SearchBar from './components/SearchBar';
 import Card from './components/Card';
 import './Folder.css';
 
-const Folder = ({items}) => {
-  if(!items || !items.links) {
-    return (
-      <div>파트 1에서 만들었던 랜딩 페이지 적용</div>
-    )
+const FOLDER_EDITS = Object.freeze([
+  { src: shareImg, name: "공유" },
+  { src: renameImg, name: "이름변경" },
+  { src: deleteImg, name: "삭제" },
+]);
+
+const Folder = () => {
+  const [folderLists, setFolderLists] = useState([]); // 이건 부모 컴포넌트에서 넘겨주는 게 좋을 듯
```

Comment:
-List 자체가 복수개의 데이터를 나타내기 때문에 추가적으로 -s는 필용벗을 것 같아요.
```suggestion
  const [folderList, setFolderList] = useState([]); // 이건 부모 컴포넌트에서 넘겨주는 게 좋을 듯
```

Reply:
사실 이부분에서 고민을 많이 했습니다. 여기서 list는 자료구조가 아닌 "폴더  목록, 항목"을 의미합니다. 따라서, 항목에는 여러 항목들이 복수로 존재한다는 의미로 접근을 했는데, 다 작성을 하고 보니, list 네이밍과 혼동을 줄 것 같다는 결론이 들더라고요.  -> 따라서, items로 변경하도록 하겠습니다. 

예상 질문 : 그럼 folders로 하면 되지 않나?
답변 : 각 항목을 클릭하면 -> 각 폴더로 접근합니다. 이 폴더를 folders라는 state에 보관해 렌더링을 진행합니다. 따라서, 이와 구별하기 위해 이렇게 네이밍을 했습니다..

Reply:
컴포넌트를 FolderList로 한 관계로, folders -> folderItems로 변경하는 것이 더 좋아 보입니다.

Reply:
좋습니다~